web/src/helpers/server/get-first-team-id-by-user-id/graphql/getTeamsByUserId.graphql

```diff
@@ -0,0 +1,5 @@
+query GetTeamsByUserId($user_id: String!) {
```

Comment:
given we're only using to fetch the default team for the user, should we add a limit of `1` (and rename the query) for improved performance?

Reply:
Done